Cabal/src/Distribution/Compat/ResponseFile.hs

```diff
@@ -65,6 +65,14 @@ escape cs c
 
 #endif
 
+-- | The arg file / response file parser.
+--
+-- This is not a well-documented capability,
+-- and is a bit eccentric (try cabal @foo @foo
+-- to see what it does), but is crucial
```

Comment:
```
f@x270:~$ cabal @foo @foo
"?"
"?"
Error: cabal: unrecognised command: @foo (try --help)
Maybe you meant `info`?
```

How is this related to line-lenght limitations?

Reply:
That should probably be documented for people who don't know about response files.